src/main/java/core/basesyntax/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport(Storage storage) {
+        StringBuilder report = new StringBuilder("fruit,quantity\n");
+        for (Map.Entry<String, Integer> entry : storage.getAllFruits().entrySet()) {
+            report.append(entry.getKey()).append(",").append(entry.getValue()).append("\n");
+        }
+        return report.toString();
+    }
+}

```

Comment:
The `ReportGeneratorImpl` class is in the default package. According to the checklist, it is recommended to organize your classes into packages with a specific purpose. Consider moving this class to a package that better reflects its role, such as `service.impl`.